### ISSUE TITLE
cannot debug with gdb-stub

### ISSUE THREAD

**Issue by lometsj** [Vita3K/Vita3K#1984]
![7(6`IP O52 $83}6NDWZ(B9](https://user-images.githubusercontent.com/7191056/181876316-396f8a1a-ce59-4112-b0eb-0ea7b9aa7cb2.gif)

**Comment by EXtremeExploit** [comment]
Indeed, currently it is broken, there is a PR that aims to fix it, https://github.com/Vita3K/Vita3K/pull/1678
